ProjectSource/src/main/java/com/kayako/sdk/messenger/rating/PostRatingBodyParams.java

```diff
@@ -1,39 +1,52 @@
 package com.kayako.sdk.messenger.rating;
 
-/**
- * @author Neil Mathew (neil.mathew@kayako.com)
- * @date 10/02/17
- */
+import java.util.HashMap;
+import java.util.Map;
+
 public class PostRatingBodyParams {
 
-    // Mandatory Fields
-    private Rating.SCORE score;
+  private static final String PARM_SCORE = "score";
```

Comment:
⚠️  This snippet of code contains typos. Spelling errors make reading and understanding the source code harder - mild errors cause the reader to stumble for a fraction of a second, many of them can cause considerable delays; really bad errors can render source code completely unreadable.  

**[More details](https://confluence.devfactory.com/x/QAPvEw)**
          
## The possible solution in this case

```JAVA
private static final String PARAM_SCORE = "score";
                               ^

```
 

<a id="8a890340-50c3-11e8-b073-875130412ce4" href="none" codereview-uuid-reserved-tag />